src/probnum/diffeq/odesolver.py

```diff
@@ -12,11 +11,12 @@
 class ODESolver(ABC):
     """Interface for ODESolver."""
 
-    def __init__(self, ivp):
+    def __init__(self, ivp, order):
         self.ivp = ivp
+        self.order = order  # RK45 has order=5, IBM(q) has order=q
         self.num_steps = 0
 
-    def solve(self, firststep, steprule, **kwargs):
+    def solve(self, steprule, atol, rtol, **kwargs):
```

Comment:
I chose atol and rtol as arguments to solve, instead of parameters of steprule because in my opinion, they are not part of the step-size-selection. A single steprule should be possible to reuse with different tolerances. Thoughts?

Reply:
How do you mean "should be possible to reuse with different tolerances"? My first impression is that it could actually make sense to have atol and rtol as part of the steprule.

Reply:
atol and rtol are in steprule now